auction_keeper/main.py

```diff
@@ -58,15 +58,20 @@ def __init__(self, args: list, **kwargs):
         parser.add_argument("--eth-key", type=str, nargs='*',
                             help="Ethereum private key(s) to use (e.g. 'key_file=aaa.json,pass_file=aaa.pass')")
 
-        parser.add_argument('--cat', type=str, help="Ethereum address of the Cat contract")
-        parser.add_argument('--vow', type=str, help="Ethereum address of the Vow contract")
+        parser.add_argument('--addresses', type=str, required=True,
```

Comment:
Since it wouldn't be specific for the tests, I suppose a new `config/` folder would be appropriate.  We could offer _kovan-addresses.json_ and _testnet-addresses.json_ initially, throwing an exception if the file isn't found (for example if they pass `mainnet` prior to the release.  Users could overwrite `testnet-addresses.json` if they want to test against a custom-deployed chain.

Reply:
sounds great!

Reply:
But should this happen here or in `pymaker`?  Is this the only MCD-specific keeper?  We wouldn't need this for `market-maker-keeper`.

Reply:
Per discussion, I will implement in `pymaker` and submit as a separate PR.